assets/js/googlesitekit-subscribe-with-google-bulk-edit.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * External dependencies
+ */
+import { render } from 'react-dom';
```

Comment:
This should be imported from `@wordpress/element` instead which wraps React. In general, we should never import anything from React directly unless it's something that `@wordpress/element` does not support (in which case we should question though whether that should be used in the first place).